frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/combined-disk.ts

```diff
@@ -33,6 +34,8 @@ export class CombinedDisk {
 
   readonly persistentVolumeClaimWrapper?: PersistentVolumeClaimWrapper;
 
+  readonly pvcName: string;
```

Comment:
CombinedDisk is an entity which should store all associated entities together. This could cause drift of the information.
Can we store the original pvc in persistentVolumeClaimWrapper variable instead?